data/tribes/initialization/empire/units.lua

```diff
@@ -1165,6 +1225,18 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_quarry",
          helptexts = {
+            lore = latin_lore(
+               -- TRANSLATORS: DO NOT TRANSLATE, but you may transliterate into non-latin fonts.
+               --    Classical Latin quote meaning:
+               --    "There shall not be left here a stone upon a stone that shall not be destroyed."
+               _("Non relinquetur hic lapis super lapidem qui non destruatur."),
+               -- TRANSLATORS: Lore helptext for an empire production site: Quarry
+               --    Translation for the classical Latin quote:
+               --    "Non relinquetur hic lapis super lapidem qui non destruatur."
+               _("There shall not be left here a stone upon a stone that shall not be destroyed.")
+            ),
+            -- TRANSLATORS: Lore author helptext for an empire production site: Quarry
+            lore_author = pgettext("empire_building", "Not much of a foretelling, when there is a quarry nearby"),
```

Comment:
sorry overlooked that earlier, this lacks a person.
how about "Foretelling of an experienced stonemason"